.markdown-link-check.json

```diff
@@ -5,6 +5,9 @@
   "ignorePatterns": [
     {
       "pattern": "^https://www.notion.so/appunite"
+    },
+    {
+      "pattern": "^https://loudius.atlassian.net/browse/LD-XXX$"
```

Comment:
**info:** it needs to be added so not found placeholder link in .github/pull_request_template.md is not reported as an issue